pkgs/applications/audio/virtual-ans/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchurl
+, unzip
+, xorg
+, libGL
+, alsaLib
+, SDL2
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "virtual-ans-${version}";
+  version = "3.0.2c";
+
+  src = fetchurl {
+    url = "https://warmplace.ru/soft/ans/virtual_ans-${version}.zip";
+    sha256 = "1dd4d91yjmp7xfjcjlam8l55r4963ypjgll97n4gjvl72m1fq856";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Photoelectronic microtonal/spectral musical instrument";
+    homepage = "https://warmplace.ru/soft/ans/";
+    license = licenses.free;
```

Comment:
I'm guessing you meant `unfree` here? I don't see the source code anywhere

Reply:
So *unfree* means *open source*?

The license file states that it is freeware.

Reply:
My bad, i was under the impression that it was free as in foss. [According to the manual](https://nixos.org/nixpkgs/manual/#sec-meta-license), i think free should indeed be fine.